tasks/plugins.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
rewrite (here and above) using modern ansible style

Reply:
Didn't understand you a bit. Can you please decipher the phrase, what is "modern ansible style"?

Reply:
Use extended yaml view. Compare the following

`chdir={{ logstash_dir }}`

```
  args:
    chdir: {{ logstash_dir }}
```